doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+Documentation workflow
+######################
+
+What you need to know
++++++++++++++++++++++
+
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
+
+The documentation for NEST is contained in the ``docs`` directory within the `NEST source code repository <https://github.com/nest/nest-simulator>`_ on GitHub.
+
+Using **GitHub** allows the NEST simulator documentation to live alongside its code. Our developers benefit from smooth version control, with each release of NEST having its own documentation.
+
+.. image:: _static/img/documentation_workflow.png
```

Comment:
I find the diagonal "Sphinx" banner in the figure confusing. There are thin arrows presumably indicating flow of information, but then the banned just cuts across. If Sphinx is used both on Github and RtD, we should have two Sphinx boxes with a size similar to other elements in the figure.

Reply:
Yes, that would be clearer. I obtained the already existing image from Sciebo. @steffengraber Do you know who created this PNG file, or where the draft is located?

Reply:
@jessica-mitchell

Reply:
@sarakonradi @jessica-mitchell If we cannot clarify/fix this within a few days, I would suggest to proceed with the current figure and create a follow-up issue about improving the figure.

Reply:
@heplesser I received the original ideas and materials from @jessica-mitchell and created a new image. Sebastian L is also happy helping us from next week onward, so we can create a follow-up issue about improving the figure later on.